ros/lib/models.py

```diff
@@ -18,6 +18,8 @@ class PerformanceProfile(db.Model):
     number_of_recommendations = db.Column(db.Integer)
     system_id = db.Column(db.Integer)
     psi_enabled = db.Column(db.Boolean)
+    top_candidate = db.Column(db.String(25), nullable=True)
+    top_candidate_price = db.Column(db.Float)
```

Comment:
IIRC, the schema for `PerformanceProfile` and `PerformanceProfileHistory` is the same. To be consistent on this, we should add these fields to the history table as well.
@kgaikwad, let me know if I'm wrong.

Reply:
You are not wrong however we are not going to show the SuggestedInstaces from history table and hence it feels like information that we are not going to use.